script.io.benchmark/default.py

```diff
@@ -0,0 +1,155 @@
+ #   Copyright (C) 2017 Lunatixz
+#
+#
+# This file is part of I/O Benchmark.
+#
+# I/O Benchmark is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
```

Comment:
Sorry copy and paste error, My standard lic is GNU... parts of this plugin are forked so I left its original MIT lic info... I'll correct the issue.